CONTRIBUTING.rst

```diff
@@ -54,6 +54,19 @@ If you are proposing a feature:
 * Remember that this is a volunteer-driven project, and that contributions
   are welcome :)
 
+Requisites
+----------
+
+Linux subsystem on Win 10
```

Comment:
this is one way to do it, but certainly isn't the only way. i'd prefer to leave basic python environment setup out since there are so many ways to do it. what if we just link to http://docs.python-guide.org/en/latest/starting/installation/ and http://docs.python-guide.org/en/latest/dev/virtualenvs/ instead?